lib/ex_aws/rds.ex

```diff
@@ -426,6 +426,58 @@ defmodule ExAws.RDS do
     request(:get, "/", query_params)
   end
 
+  @doc """
+  Creates a DBSnapshot. The source DBInstance must be in "available" state.
+  See <https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBSnapshot.html>
+  """
+  @spec create_db_snapshot(instance_id :: binary, snapshot_id :: binary) :: ExAws.Operation.RestQuery.t()
+  def create_db_snapshot(instance_id, snapshot_id) do
+    query_params = %{
+      "Action" => "CreateDBSnapshot",
+      "DBInstanceIdentifier" => instance_id,
+      "DBSnapshotIdentifier" => snapshot_id,
+      "Version" => @version
+    }
+
+    request(:post, "/", query_params)
+  end
+
+
+  @type create_db_snapshot_opts :: [
```

Comment:
Maybe rename this type, and the related argument to `describe_db_snapshot_opts`?